### ISSUE TITLE
When submitting, author cannot return to first page of process

### ISSUE THREAD

**Issue by StephDriver** [openlibhums/janeway#4458]
## User Story

As an author, when I am submitting an article, I have to navigate a multipage process. I am guided through this by a chart at the top, which allows me to move back to previously completed pages, except for the first page.
<img width="1030" alt="image" src="https://github.com/user-attachments/assets/281fc150-f5fd-4612-b5c4-3829e1b0ee66">

As I'm entering information on a later page, I may realise I've made a mistake and want to check back on the first page, particularly as there is a rich text entry option on the first page, so it wasn't just a case of ticking some boxes which if not ticked I'd not be able to proceed.

## Solution I'd like
either:
Include the start page in the map, but make the tick boxes non-editable.

or:

Move the 'competing interest' text entry to a different page / new page.
make the tick boxes for the start of the submission separate from the rest of this process, and don't include it in the visual map.

---
note: 
first page URL: `/submit/start`
subsequent pages: `/submit/[article_id]/info`, `/submit/[article_id]/authors` etc. 

If there first page is just tick boxes, without agreeing to which one cannot proceed, then there is some sense in it not be editable later, but I think it should be distinguished from the rest of the pages, not part of the same map.  

Currently the first page also has a rich text box for competing interest information, this should be editable during the process. Perhaps it should move to a different page?
